make/local.example

```diff
@@ -0,0 +1,29 @@
+# This file contains examples for makefile flags that can be used in Cmdstan's make/local file
```

Comment:
[optional] Making the directions on using this more explicit might be helpful.

```
# To use this template, make a copy from make/local.example to make/local and uncomment options as needed. Be sure to run `make clean-all` before compiling a model to make sure everything gets rebuilt.
```

Something like that